### PR TITLE
Implement custom theme

### LINKED ISSUE #446: Implement custom theme for chat settings
**Actual Behaviour**
Custom theme for chat is not available in settings

**Expected Behaviour**
Implement custom theme for chat settings

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by PrP-11**
Fixes #446 

Changes: Implement custom theme for chat settings

Surge Deployment Link: https://pr-447-fossasia-susi-accounts.surge.sh

Screenshots for the change:
theme

### CODE REVIEW

File: src/components/Settings/ThemeChanger.js [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+    settingsChanged.customThemeValue = customData;
+    if (this.state.bodyBackgroundImage || this.state.messageBackgroundImage) {
+      settingsChanged.backgroundImage =
+        this.state.bodyBackgroundImage +
+        ',' +
+        this.state.messageBackgroundImage;
+    } else {
+      settingsChanged.backgroundImage = '';
+    }
+    console.log(settingsChanged);
```
Comment (DNS-404) [inline src/components/Settings/ThemeChanger.js]: Remove this.

**Comment by PrP-11** [comment]
@DNS-404 The value of background color is changing successfully on the server and you can verify it using the console statement above. But it seems that chat.susi is not fetching the changes. I have opened https://github.com/fossasia/chat.susi.ai/issues/1533 to fix that problem

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by PrP-11** [comment]
@mariobehling Resoved the conflicts and implemented a basic version of color picker. Please review.

**Pull request merged**